docs/tutorials/11_deuteron_binding_energy.ipynb

```diff
@@ -0,0 +1,1374 @@
+    "$a_n \\ \\rightarrow \\ \\frac{1}{2}\\left[\\prod_{j=0}^{n-1}-Z_j\\right](X_n+iY_n).$\n",
+    "\n",
+    "Fortunately, in Qiskit Nature we have tools to define fermionic operators and to convert this kind of operators into Pauli operators through the Jordan-Wigner transformation. Initially, we have to define an empty variable called hamiltonian, using the zero() method of the object FermionicOp. After this, we use a for loop to create the string label that is used to create the FermionicOp that we will use in matrix elements of kinetic and potential energies. Then, we use each matrix element to update our initial hamiltonian variable. At the end of the for loop, we have in hands a Hamiltonian in terms of the creation and annihilation operators, so we need to use JordanWignerMapper() and the QubitConverter() to complete the task of obtain the Deuteron Hamiltonian in terms of Pauli operators. In order to obtain more information about the Qiskit Nature tools, we recommend a reading of the Qiskit documentation [3]. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "dae85350",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def create_deuteron_hamiltonian(N: int):\n",
+    "    \"\"\"Creates a version of the Deuteron Hamiltonian in terms of operators\n",
+    "    compatible with qubits.\n",
+    "\n",
+    "        Args:\n",
+    "            N (int): An integer number that represents the dimension of the\n",
+    "            basis.\n",
+    "\n",
+    "        Returns:\n",
+    "            A Hamiltonian converted to qubits that represents the Deuteron.\n",
+    "    \"\"\"\n",
+    "    if type(N) == int:\n",
+    "        hamiltonian = FermionicOp.zero(N)\n",
+    "\n",
+    "        for i in range(N):\n",
+    "            for j in range(N):\n",
+    "\n",
+    "                label = \"+_{} -_{}\".format(str(i), str(j))\n",
+    "                hamiltonian += (\n",
+    "                    3.5\n",
+    "                    * (\n",
+    "                        (2 * j + 3 / 2) * kronecker_delta_function(i, j)\n",
+    "                        - np.sqrt(j * (j + (1 / 2))) * kronecker_delta_function(i + 1, j)\n",
+    "                        - np.sqrt((j + 1) * (j + (3 / 2)) * kronecker_delta_function(i - 1, j))\n",
+    "                    )\n",
+    "                    * FermionicOp(label, display_format=\"sparse\")\n",
+    "                )\n",
+    "                hamiltonian += (\n",
+    "                    -5.68658111\n",
+    "                    * kronecker_delta_function(0, j)\n",
+    "                    * kronecker_delta_function(i, j)\n",
+    "                    * FermionicOp(label, display_format=\"sparse\")\n",
+    "                )\n",
+    "\n",
+    "        mapper = JordanWignerMapper()\n",
+    "        qubit_converter = QubitConverter(mapper=mapper, two_qubit_reduction=True)\n",
+    "        qubit_hamiltonian = qubit_converter.convert_match(hamiltonian)\n",
```

Comment:
```suggestion
    "        qubit_hamiltonian = qubit_converter.convert(hamiltonian)\n",
```
`convert_match` is only useful if another operator has already been converted by
this converter instance. Since this is clearly not the case here, you should use
`convert` instead.